packages/react-hooks/src/mixer/useMixer.ts

```diff
@@ -28,10 +30,19 @@ export const useMixerGroups = (): MixerAssetGroup[] => {
   }, [mixerGroupIds]);
 };
 
+export const useMixerInfos = (): [StorageKey, MixerInfo][] => {
+  const mixerGroups = useCall<Array<[StorageKey, MixerInfo]>>('query.mixer.mixerGroups.entries', [], undefined, []);
+
+  return useMemo(() => {
+    return mixerGroups || [];
+  }, [mixerGroups]);
+};
+
 const logger = LoggerService.new('MixerUsage');
 
 export const useMixer = () => {
-  const mixerAssetGroup = useMixerGroups();
+  const mixerIds = useMixerGroups();
+  const mixerInfos = useMixerInfos();
 
```

Comment:
I'm not sure if we need `mixerInfos` for the mixer hook, `useMixer` should contain the code required for initing the mixer.

`useMixerInfos` can go to the context if caching is needed as it will be created one time, or it can be called directly.

Reply:
So mixer infos contains the details about the amounts each mixer requires. So we need it to display the mixer selections (by amounts).